setup.py

```diff
@@ -159,7 +159,7 @@ def _is_debug_build():
           setup_requires=["wheel", "cachetools", "pefile"],
           install_requires=["cachetools", "pefile"],
           platforms="Windows",
-          python_requires='>=3.7, <3.12',
+          python_requires='>=3.7, <3.13',

```

Comment:
To solve a conflict with latest `master`, please bump the minimum version to 3.8